plugins/webp-uploads/helper.php

```diff
@@ -411,6 +411,17 @@ function webp_uploads_is_fallback_enabled(): bool {
 	return (bool) get_option( 'perflab_generate_webp_and_jpeg' );
 }
 
+/**
+ * Checks if the `perflab_generate_all_fallback_sizes` option is enabled.
+ *
+ * @since n.e.x.t
+ *
+ * @return bool True if the option is enabled, false otherwise.
+ */
+function webp_uploads_should_generate_all_fallback_sizes(): bool {
+	return (bool) get_option( 'perflab_generate_all_fallback_sizes' );
```

Comment:
Should add default value here as well.

```suggestion
	return (bool) get_option( 'perflab_generate_all_fallback_sizes', 0 );
```